cob_hardware_config/common/Elmo.dcf

```diff
@@ -1327,6 +1327,7 @@ ObjectType=0x7
 DataType=0x0007
 AccessType=rw
 PDOMapping=0
+ParameterValue=0x20410020
 
 [1A00sub4]
 ParameterName=PDO Mapping Entry_4
```

Comment:
@ipa-mdl 
please verify the dcf contains your latest valid configuration/changes/values